book/C-git-commands/1-git-commands.asc

```diff
@@ -234,124 +234,133 @@ Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorat
 Sa <<_distributed_git#_private_team>> at <<_git_tools#_commit_ranges>> aming tinalakay ang `branchA..branchB` na syntax para gamitin ang `git log` na utos para makita kung ano ang kaibahan sa isang katugong branch sa iba pang branch.
 Sa <<_git_tools#_commit_ranges>> malawak naming tinalakay ito.
 
-In <<_git_tools#_merge_log>> and <<_git_tools#_triple_dot>> we cover using the `branchA...branchB` format and the `--left-right` syntax to see what is in one branch or the other but not in both.
-In <<_git_tools#_merge_log>> we also look at how to use the `--merge` option to help with merge conflict debugging as well as using the `--cc` option to look at merge commit conflicts in your history.
+Sa <<_git_tools#_merge_log>> at <<_git_tools#_triple_dot>> tinalakay natin sa `branchA...branchB` format at ang `--left-right` na syntax para makita kung ano ang nasa isang branch o sa ibang branch pero hindi sa dalawa.
```

Comment:
Please change ``sa `branchA...branchB` format`` to ``ang `branchA...branchB` na format``